tests/components/smartthings/test_climate.py

```diff
@@ -330,20 +325,26 @@ def air_conditioner_fixture(device_factory):
     assert state.attributes["drlc_status_level"] == -1
     assert state.attributes["drlc_status_start"] == "1970-01-01T00:00:00Z"
     assert state.attributes["drlc_status_override"] is False
-    assert state.attributes["power_consumption_start"] == "2019-02-24T21:03:04Z"
+    assert state.attributes[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)